tests/ops/functions/test_matrix.py

```diff
@@ -659,6 +659,34 @@ def circuit(theta):
 
         assert np.allclose(matrix, expected_matrix)
 
+    @pytest.mark.catalyst

```

Comment:
Doesn't look like we have a `catalyst` marker defined yet for our test suite. Might be time to add one to `tests/pytest.ini` though.

Reply:
Good idea, added!